### ISSUE TITLE
DO NOT UPDATE TO 22.37 (broken Error 403)

### ISSUE THREAD

**Issue by powerhousepr** [yuliskov/SmartTube#3622]
The App just updated to the latest log 22.37 and now it no longer works giving error 403 in all videos. It was just fine before the update.

**Comment by Ramshu** [comment]
Unfortunately I updated and now smartube doesn't play anything.

**Comment by DiscoFever78** [comment]
Error 403 is occurring on prior versions as well.

**Comment by josejoa** [comment]
I updated also :-(

**Comment by tbolick** [comment]
Seems to be happening on 22.09 as well.  DID NOT UPDATE.  So it may be a bigger issue.  Does it work for anyone?

**Comment by powerhousepr** [comment]
> Error 403 is occurring on prior versions as well.

NEVER like this, it was random and it would reload itself. No nothing plays.

**Comment by liberodark** [comment]
same for me error 403

**Comment by MurphyDoe0815** [comment]
error 403

**Comment by alexanderbont** [comment]
Same problem here, is there any possibly to revoke that update, or how fast can this be fixed?
Now I can't use smart tube at all

**Comment by tidder99** [comment]
Does nobody check for other posts before making a new one 
There's like five of them saying the same thing

**Comment by tbolick** [comment]
Can anyone confirm it works for ANYONE at any update level?  I'm thinking Google screwed it up.

**Comment by powerhousepr** [comment]
> error 403



> Can anyone confirm it works for ANYONE at any update level? I'm thinking Google screwed it up.

YES, Me… it was working just fine all day until I updated just now.

**Comment by powerhousepr** [comment]
> Does nobody check for other posts before making a new one There's like five of them saying the same thing

NONE are advising to avoid the update in the header.

**Comment by powerhousepr** [comment]
> Same problem here, is there any possibly to revoke that update, or how fast can this be fixed? Now I can't use smart tube at all

I DELETED it and “re-download the 22.36” but when you open it is just the same broken 22.37 version. This is crazy. Might need to look for an older version in another website.

**Comment by mcrypt** [comment]
revanced still works on the phone so not every non native client seems to have this problem..

**Comment by oregonpillow** [comment]
Broke for me too since updating.
Went to settings, shocked there is no rollback ability. 

⚠️‼️feature request: Rollback to previous version should be the next feature implemented after a 403 hot fix. It’s an absolute must imo.

**Comment by Memmopaco** [comment]
A me sta funzionando solo la parte live il resto no

**Comment by liberodark** [comment]
For my previous version and this have 403 error

**Comment by Sector6759** [comment]
Duplicate of https://github.com/yuliskov/SmartTube/issues/3606

**Comment by powerhousepr** [comment]
> A me sta funzionando solo la parte live il resto no

A mi no me funciona nada.

**Comment by domdal** [comment]
> Broke for me too since updating.
> 
> Went to settings, shocked there is no rollback ability. 
> 
> 
> 
> ⚠️‼️feature request: Rollback to previous version should be the next feature implemented after a 403 hot fix. It’s an absolute must imo.
> 
> 
> 
> 

Stable 22.19 is also not working

**Comment by liberodark** [comment]
Probably is better to closing this issue because is duplicated.
We need to wait for find a workaround about this error.
Probably Googe have change something on the API.

**Comment by purplescorpion1** [comment]
> Broke for me too since updating. Went to settings, shocked there is no rollback ability.
> 
> ⚠️‼️feature request: Rollback to previous version should be the next feature implemented after a 403 hot fix. It’s an absolute must imo.

There is no such thing as a rollback on android. Each app has a version number and a signature. You can only install app which is equal to or higher in version number with the same signature. (unless you have rooted and disabled version and signature checks)

And previous versions are effected anyway - The new versions were trying to fix it

**Comment by ACR-Jeff** [comment]
This isn't an issue with only the newest update, It is an issue with youtube entirely messing with stuff on their end, I have had both older version of stable and newer update bata on my system and they both have the same 403 error

**Comment by gamepro9000** [comment]
this is probably because google is expirimenting with puting ads directly into the video by making it apart of em dus surpasing add blockers etc

**Comment by deucetresthugz** [comment]
> Same problem here, is there any possibly to revoke that update, or how fast can this be fixed?
> 
> Now I can't use smart tube at all

Ive tried installing older releases on my Nvidia Shield however now they're all giving me 403 errors for some strange reason. The previous update from yesterday before this latest one worked beautifully but this one has caused some other lasting issues as stated above.

**Comment by domdal** [comment]
> > > Broke for me too since updating. Went to settings, shocked there is no rollback ability.
> 
> > > ⚠️‼️feature request: Rollback to previous version should be the next feature implemented after a 403 hot fix. It’s an absolute must imo.
> 
> > 
> 
> > There is no such thing as a rollback on android. Each app has a version number and a signature. You can only install app which is equal to or higher in version number with the same signature. (unless you have rooted and disabled version and signature checks)
> 
> > 
> 
> > And previous versions are effected anyway - The new versions were trying to fix it
> 
> 
> 
> NO, previous versions were not affected. I was LITERALLY using it just a few minutes ago just fine. We are not talking about the RANDOM ERRO 403. We are talking about a constant NOT working app problem now displaying error 403.

I just tried different older versions back to 21.48 and none of them is working

**Comment by deucetresthugz** [comment]
> > Same problem here, is there any possibly to revoke that update, or how fast can this be fixed? Now I can't use smart tube at all
> 
> 
> 
> I DELETED it and “re-download the 22.36” but when you open it is just the same broken 22.37 version. This is crazy. Might need to look for an older version in another website.

Ive went back as far as 22.19 stable but the 403 issue persists.

**Comment by gamepro9000** [comment]
> > > Same problem here, is there any possibly to revoke that update, or how fast can this be fixed? Now I can't use smart tube at all
> > 
> > 
> > I DELETED it and “re-download the 22.36” but when you open it is just the same broken 22.37 version. This is crazy. Might need to look for an older version in another website.
> 
> Ive went back as far as 22.19 stable but the 403 issue persists.

google just ruined it probably with there new ad system

**Comment by gamepro9000** [comment]
yes its google  new pipe doesnt work either whole difrent app also add blocker youtube

**Comment by gamepro9000** [comment]
i was afraid this is gonna hapen

**Comment by terchitui** [comment]
even 18.26 version does not work

**Comment by josejoa** [comment]
If the older versions also don't work, then is Google for sure.

**Comment by jmhg92** [comment]
I am also getting the 403 error.
My app just updated to 22.37 a few minutes ago and now EVERY video I try playing gives me the error, even the ones I watched today before the update.

**Comment by gamepro9000** [comment]
its because now when you watch a video on youtube it sends the video to a server where ads are inserted into the video thus ad blockers etc dont work bc they will get detected oe the vid will play ans you will still have ads or it wont work at all

**Comment by ACR-Jeff** [comment]
> > > Broke for me too since updating. Went to settings, shocked there is no rollback ability.
> > > ⚠️‼️feature request: Rollback to previous version should be the next feature implemented after a 403 hot fix. It’s an absolute must imo.
> > 
> > 
> > There is no such thing as a rollback on android. Each app has a version number and a signature. You can only install app which is equal to or higher in version number with the same signature. (unless you have rooted and disabled version and signature checks)
> > And previous versions are effected anyway - The new versions were trying to fix it
> 
> NO, previous versions were not affected. I was LITERALLY using it just a few minutes ago just fine. We are not talking about the RANDOM ERRO 403. We are talking about a constant NOT working app problem now displaying error 403.

I disagree, It isn't a random app problem, It is also Freetube, Newpipe and other apps having this issue, It is a youtube issue.

**Comment by ChrisXpert** [comment]
yes, I upgraded because I had this 403 error today, and it doesn't work with the new version either! Blocked by Google for sure

**Comment by gamepro9000** [comment]
i hope they find a way around it or il kms

**Comment by ACR-Jeff** [comment]
For me Live streams are still working

**Comment by gamepro9000** [comment]
> For me Live streams are still working

il check if it works here too

**Comment by josejoa** [comment]
live streams don't have ads

**Comment by mcrypt** [comment]
Well ReVanced still works for mobile so it clearly can be done

**Comment by gamepro9000** [comment]
> For me Live streams are still working

works fine too so it for sure is in the ads part

**Comment by 7xdevs** [comment]
> I disagree, It isn't a random app problem, It is also Freetube, Newpipe and other apps having this issue, It is a youtube issue.
YMusic is working perfectly with no ads. Not sure how they do it.

**Comment by ACR-Jeff** [comment]
Wondering if this is related, I've noticed ever since this started my pi-hole adblocker has been spammed tons by `embeddedassistant.googleapis dot com`

**Comment by tidder99** [comment]
I have one fire stick working. Not been updated since 21.91s
And one not working fire stick which I updated a couple hours ago
Which is weird if it's a YouTube server thing 
Even if I uninstall and install the older version, it still doesn't work 
They are both logged into the same account

**Comment by gamepro9000** [comment]
> I have one fire stick working. Not been updated since 21.91s And one not working fire stick which I updated a couple hours ago Which is weird if it's a YouTube server thing Even if I uninstall and install the older version, it still doesn't work They are both logged into the same account

weird  i gues we should just wait for next update and let the devs figure it out maybe they will give a explanation

**Comment by dontwantonebutneedone** [comment]
> > Same problem here, is there any possibly to revoke that update, or how fast can this be fixed? Now I can't use smart tube at all
> 
> I DELETED it and “re-download the 22.36” but when you open it is just the same broken 22.37 version. This is crazy. Might need to look for an older version in another website.

Swell. I should have read further before trying to do the same and getting the same result. I guess I'll wait to see how this all shakes out. Not working on most videos, but a few will still play here and there.

**Comment by powerhousepr** [comment]
> I have one fire stick working. Not been updated since 21.91s And one not working fire stick which I updated a couple hours ago Which is weird if it's a YouTube server thing Even if I uninstall and install the older version, it still doesn't work They are both logged into the same account

THAT is what I am saying, the previous build was working just fine. People in here using SUPER OLD versions are obviously not going to work but the more recent ones like 22.35, 22.34 etc should work fine. Mine was working all day until I updated.

**Comment by domdal** [comment]
> > I have one fire stick working. Not been updated since 21.91s And one not working fire stick which I updated a couple hours ago Which is weird if it's a YouTube server thing Even if I uninstall and install the older version, it still doesn't work They are both logged into the same account
> 
> 
> 
> THAT is what I am saying, the previous build was working just fine. People in here using SUPER OLD versions are obviously not going to work but the more recent ones like 22.35, 22.34 etc should work fine. Mine was working all day until I updated.


No version is working
I've tested different including the last view betas

**Comment by EdmundGerber** [comment]
> > I have one fire stick working. Not been updated since 21.91s And one not working fire stick which I updated a couple hours ago Which is weird if it's a YouTube server thing Even if I uninstall and install the older version, it still doesn't work They are both logged into the same account
> 
> THAT is what I am saying, the previous build was working just fine. People in here using SUPER OLD versions are obviously not going to work but the more recent ones like 22.35, 22.34 etc should work fine. Mine was working all day until I updated.

So was mine, using 22.35. But now it's nothing but 403 error, despite not updating. So it's something Google is doing on their end, and nothing to do with any updates of Smarttube

**Comment by josejoa** [comment]
> > > I have one fire stick working. Not been updated since 21.91s And one not working fire stick which I updated a couple hours ago Which is weird if it's a YouTube server thing Even if I uninstall and install the older version, it still doesn't work They are both logged into the same account
> > 
> > 
> > THAT is what I am saying, the previous build was working just fine. People in here using SUPER OLD versions are obviously not going to work but the more recent ones like 22.35, 22.34 etc should work fine. Mine was working all day until I updated.
> 
> So was mine, using 22.35. But now it's nothing but 403 error, despite not updating. So it's something Google is doing on their end, and nothing to do with any updates of Smarttube

Always there is a bypass ;)

**Comment by deucetresthugz** [comment]
> > > I have one fire stick working. Not been updated since 21.91s And one not working fire stick which I updated a couple hours ago Which is weird if it's a YouTube server thing Even if I uninstall and install the older version, it still doesn't work They are both logged into the same account
> 
> > 
> 
> > THAT is what I am saying, the previous build was working just fine. People in here using SUPER OLD versions are obviously not going to work but the more recent ones like 22.35, 22.34 etc should work fine. Mine was working all day until I updated.
> 
> 
> 
> So was mine, using 22.35. But now it's nothing but 403 error, despite not updating. So it's something Google is doing on their end, and nothing to do with any updates of Smarttube
> 
> 

I don't think that's the case bc i have a jailbroken iphone with a free ad free youtube tweak that's still working flawlessly. If it was a server side change then i'm not sure how my tweak would still work ad free?

**Comment by EdmundGerber** [comment]
> > > > I have one fire stick working. Not been updated since 21.91s And one not working fire stick which I updated a couple hours ago Which is weird if it's a YouTube server thing Even if I uninstall and install the older version, it still doesn't work They are both logged into the same account
> > 
> > 
> > > 
> > 
> > 
> > > THAT is what I am saying, the previous build was working just fine. People in here using SUPER OLD versions are obviously not going to work but the more recent ones like 22.35, 22.34 etc should work fine. Mine was working all day until I updated.
> > 
> > 
> > So was mine, using 22.35. But now it's nothing but 403 error, despite not updating. So it's something Google is doing on their end, and nothing to do with any updates of Smarttube
> 
> I don't think that's the case bc i have a jailbroken iphone with a free ad free youtube tweak that's still working flawlessly. If it was a server side change then i'm not sure how my tweak would still work ad free?

We can't say as it's might be like comparing apples to oranges. You should get in touch with the author of this software, though, and let him know you might be able to contribute to a fix.

**Comment by gramsaran** [comment]
> Does nobody check for other posts before making a new one There's like five of them saying the same thing

Poor mods

**Comment by ICE0124** [comment]
The update didn't break anything for me, i went to a different TV with an older version and it didnt work then i went to another with an even older version and got the same error. The update didnt do anything youtube did on their end.

**Comment by dontwantonebutneedone** [comment]
I put a free trial of YT premium on the acct and now the current version works fine. So it looks like something they did. Some videos would play (news and live tv) without premium while another tv I didn't update kept working fine throughout. Not sure if its targeted or random.

They're offering 1 and 2 month free trials of YT. I saw 2 months just going to youtube and getting a popup on the main page. If you just go look at premium manually, they only give you one month. So this can sit for a few months for me while they figure it out.

Other thing that is different in a good way after getting premium is that smarttube used to cut the last 2-3 seconds off of videos and a lot of clips I watch people talk to the very end, so the last word disappeared. That's not a thing after subbing to premium.

**Comment by devcomputer** [comment]
apps, smartphone and classic internet have all well passed the zenith.
googles last service people seem to like is gmail. the amount of redundant apps on any smartphone is a waste of time and resources and who needs smartphones when you have an ai assistant.

**Comment by Sarafian** [comment]
I updated yesterday. Everything was working. After the update it's not.  So not sure if previous versions have the same issue.

**Comment by aekramer** [comment]
> Can anyone confirm it works for ANYONE at any update level? I'm thinking Google screwed it up.

No google did not mess it up, it is most likely an intended update on their part to break 3rd party viewing apps that prevent ads.

**Comment by 2fast4u-gmc** [comment]
Hurray, with update 22.38 it's working again.
Many thanks to the developers!
But guys, keep calm. The error was not due to Smart Tube but YouTube blocks 3rd party apps.

**Comment by Sarafian** [comment]
> > Can anyone confirm it works for ANYONE at any update level? I'm thinking Google screwed it up.
> 
> No google did not mess it up, it is most likely an intended update on their part to break 3rd party viewing apps that prevent ads.

Not sure why everyone uses the word `screw up` for a business that is trying to protect their income with very intentional moves. I don't like it eather, but it is what it is. I just don't get how they do it, because in a browser session without sign in, you can still watch the videos. It's more and more hidden but it doesn't stop you.

SmartTube uses, user context so probably it's much easier for google.

**Comment by josejoa** [comment]
Maybe we can close this thread after the update, thanks to everybody.

**Comment by powerhousepr** [comment]
UPDATE 22.38 makes the App usable again up to 1080p. 

In the previous builds, most of the random Error 403 would Pop up for me ONLY when playing 4k content. So this makes sense thats only up to 1080p for the moment.

i will close this, make sure to come back here before making ANY updates from now on.

Thank you all.

**Comment by CoreOnCrack** [comment]
It is NOT the update all versions are effected.

What you should not do is force stop and delete data, that will remove all your settings.

Go make a backup and wait for a update with a fix  GL peeps

**Comment by ChrisXpert** [comment]
Yes UPDATE 22.38 makes the App usable again up to 1080p. Tysm !

**Comment by CoreOnCrack** [comment]
yuliskov is a freaking HERO!

**Comment by DomHG** [comment]
Yuslikov Super Top Mega Cool 🌴

**Comment by Thomalabars** [comment]
I have another streaming service on my TV (onstream) and it's also not working so I think maybe Google just did some stuff to break all the not officials streaming services?

**Comment by hjkgugy345** [comment]
I'm using version 22.38. Vids are played OK, but only AVC format is available and accessible to play. Is it normal?
